web-features.json

```diff
@@ -0,0 +1,300 @@
+      "overscroll-behavior",
+      "scroll-snap",
+      "scroll-behavior"
+    ],
+    "interop-2022-subgrid": [
+      "subgrid"
+    ],
+    "interop-2021-transforms": [
+      "transforms2d",
+      "transforms3d"
+    ],
+    "interop-2023-url": [
+      "url"
+    ],
+    "interop-2023-webcodecs": [
+      "webcodecs"
+    ],
+    "interop-2023-webcompat": [],
+    "interop-2023-webcomponents": [
+      "constructed-stylesheets",
+      "form-associated-custom-elements"
+    ]
+  },
+  "2024": {
+    "interop-2024-accessibility": [],
+    "interop-2024-nesting": [
+      "nesting"
+    ],
+    "interop-2023-property": [
+      "registered-custom-properties"
+    ],
+    "interop-2024-dsd": [
+      "declarative-shadow-dom"
+    ],
+    "interop-2024-font-size-adjust": [
+      "font-size-adjust"
+    ],
+    "interop-2024-websockets": [],
+    "interop-2024-indexeddb": [
+      "indexeddb"
+    ],
+    "interop-2024-layout": [
+      "flexbox",
+      "grid",
+      "subgrid"
+    ],
+    "interop-2023-events": [
+      "mouse-events",
+      "pointer-events-api"
+    ],
+    "interop-2024-popover": [
+      "popover"
+    ],
+    "interop-2024-relative-color": [
+      "relative-color"
+    ],
+    "interop-2024-video-rvfc": [
+      "request-video-frame-callback"
+    ],
+    "interop-2024-scrollbar": [
+      "scrollbar-gutter",
+      "scrollbar-width"
+    ],
+    "interop-2024-starting-style-transition-behavior": [
+      "starting-style",
+      "transition-behavior"
+    ],
+    "interop-2024-dir": [],
+    "interop-2024-text-wrap": [
+      "text-wrap",
+      "text-wrap-balance"
+    ],
+    "interop-2023-url": [
+      "url"
+    ]
+  },
+  "2025": {
+    "interop-2025-backdrop-filter": [
+      "backdrop-filter"
+    ],
+    "interop-2025-core-web-vitals": [
+      "largest-contentful-paint"
+    ],
+    "interop-2025-anchor-positioning": [
+      "anchor-positioning"
+    ],
+    "interop-2025-details": [
+      "details"
+    ],
+    "interop-2024-layout": [
+      "flexbox",
+      "grid",
+      "subgrid"
+    ],
+    "interop-2025-modules": [
+      "json-modules"
+    ],
+    "interop-2025-navigation": [
+      "navigation"
+    ],
+    "interop-2023-events": [
+      "pointer-events-api",
+      "mouse-events"
+    ],
+    "interop-2025-remove-mutation-events": [
+      "mutation-events"
+    ],
+    "interop-2025-scope": [
+      "scope"
+    ],
+    "interop-2025-scrollend": [
+      "scrollend"
+    ],
+    "interop-2025-storageaccess": [
+      "storage-access"
+    ],
+    "interop-2025-textdecoration": [
+      "text-decoration"
+    ],
+    "interop-2025-urlpattern": [
+      "urlpattern"
+    ],
+    "interop-2025-view-transitions": [
+      "view-transitions",
+      "view-transition-class"
+    ],
+    "interop-2025-webassembly": [
+      "wasm-string-builtins"
+    ],
+    "interop-2025-webcompat": [
+      "appearance",
+      "zoom",
+      "list-style"
+    ],
+    "interop-2025-webrtc": [
+      "webrtc",
```

Comment:
This feature has a much larger scope than the Interop 2025 focus area. I think we're probably best mapping to only webrtc-encoded-transform.